front/pages/w/[wId]/builder/assistants/create.tsx

```diff
@@ -182,7 +183,15 @@ export default function CreateAssistant({
         <Page variant="modal">
           <div className="flex flex-col gap-6 pt-9">
             <div className="flex min-h-[20vh] flex-col justify-end gap-6">
-              <Page.Header title="Start new" icon={PencilSquareIcon} />
+              <div className="flex flex-row items-center gap-2">
```

Comment:
Same for the editing tool configuration, I feel weird to see the icon on top of the title, I think it's because it's not really a page? 

before:
<img width="969" alt="Screenshot 2025-05-14 at 21 31 07" src="https://github.com/user-attachments/assets/67e73df5-e604-4a28-b339-7f30cb9c7588" />


after:
<img width="958" alt="Screenshot 2025-05-14 at 21 30 50" src="https://github.com/user-attachments/assets/5f4b0c8e-1b80-4580-a72e-aac6fba4109b" />